configure.sh

```diff
@@ -43,12 +43,13 @@ esac
 pip install $QUIET_FLAG -r requirements.txt
 
 TF_CFLAGS=( $(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_compile_flags()))') )
-TF_LFLAGS=( $(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_link_flags()))') )
-TF_SHAREDLIB=( $(python -c 'import tensorflow as tf; print(tf.sysconfig.get_link_flags()[-1])') )
+TF_LFLAGS="$(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_link_flags()))')"
 TF_CXX11_ABI_FLAG=( $(python -c 'import tensorflow as tf; print(tf.sysconfig.CXX11_ABI_FLAG)') )
```

Comment:
On my version `tf.sysconfig.CXX11_ABI_FLAG` does not exist. 
From TF v1.9 to 1.13 [I use `tf.__cxx11_abi_flag__`](https://github.com/PatWie/tensorflow-cmake/blob/dc0b4eeee274849abc1effdb29df6b3cd5f39d06/cmake/modules/FindTensorFlow.cmake#L65)

Reply:
Thank you, Patrick. Addons is expected to be compatible with tf 2.x only.